website/docs/r/tg_gateway.html.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: "ibm"
+page_title: "IBM : tg_gateway"
+sidebar_current: "docs-ibm-resource-tg-gateway"
+description: |-
+  Manages IBM Transit Gateway.
+---
+
+# ibm\_tg_gateway
+
+Provides a transit gateway resource. This allows transit gateway to be created, and updated and deleted.
+
+## Example Usage
+
+```hcl
+resource "ibm_tg_gateway" "new_tg_gw"{
+name="transit-gateway-1"
+location="us-south"
+global=true
+resource_group=""
```

Comment:
don't send empty rsg grp....send some value or remove this

Reply:
added resource_group parameter as well.